core/server/data/schema/schema.js

```diff
@@ -459,5 +459,16 @@ module.exports = {
         created_by: {type: 'string', maxlength: 24, nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    email_recipients: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        email_id: {type: 'string', maxlength: 24, nullable: false, references: 'emails.id'},
+        member_id: {type: 'string', maxlength: 24, nullable: false, index: true},
+        batch: {type: 'integer', nullable: false, unsigned: true, index: true},
+        processed_at: {type: 'dateTime', nullable: true},
+        // TODO: should these be prefixed with `member_` to denote their origin?
+        uuid: {type: 'string', maxlength: 36, nullable: false, validations: {isUUID: true}},
+        email: {type: 'string', maxlength: 191, nullable: false, validations: {isEmail: true}},
+        name: {type: 'string', maxlength: 191, nullable: true}
```

Comment:
@matthanley the `member_` prefix wasn't in the original db schema in https://github.com/TryGhost/Team/issues/353#issue-668959800 but I think it would be useful, especially for the `uuid` field because that could be quite confusing if you expect it to be a uuid but it's not unique across `email_recipients` rows. Do you think it's a change worth making?